profile/profile_test.go

```diff
@@ -108,6 +108,22 @@ func TestParseError(t *testing.T) {
 	}
 }
 
+func TestParseConcatentated(t *testing.T) {
+	prof := testProfile1.Copy()
+	prof.TimeNanos = 10000
```

Comment:
Perhaps just add this to the testProfile1 initialization? In general every profile should have TimeNanos set, so it's a bit strange to just have it initialized here.

Reply:
Done.